packages/create-cloudflare/src/templates.ts

```diff
@@ -385,12 +385,18 @@ export const createContext = async (
 			value: "web-framework",
 			description: "Select from the most popular full-stack web frameworks",
 		},
-		{
-			label: "Application Starter",
-			value: "demo",
-			description:
-				"Select from a range of starter applications using various Cloudflare products",
-		},
+		...(args.experimental
```

Comment:
Could we be more sophisticated here and check to see if each of these categories has any items? Rather than hard coding it?

Reply:
yeah that's a good suggestion, unfortunately the code structure makes that quite challenging since the category options are defined here and then the various options for them are computed on demand in the `if-else` statements below based on the user's choice. Filtering out he category options based on their available options would require a good amount of code to be moved around I think.

I am not sure if such a small issue (that only applies to experimental mode) would justify the non trivial refactoring that I think would be necessary here 🤔